### PR TITLE
Components tests

### PR DESCRIPTION
**Pull request by wackydapper**

### CODE REVIEW

**Comment by wackydapper** [comment]
[UI: Add unit tests for components](https://trello.com/c/IkneoMgA/53-ui-add-unit-tests-for-components)

File: static/src/components/PlayerDisplay.js [hunk 1]
```diff
@@ -7,7 +7,6 @@ import { bindActionCreators } from 'redux';
 import * as actions from '../actions/game';
 import Inventory from './Inventory';
 import Wonder from './Wonder';
-import CardHist from './CardHist';
```
Comment (mmdonaldson) [inline static/src/components/PlayerDisplay.js]: I don't think you mean to delete this?

File: static/src/components/PlayerDisplay.js [hunk 2]
```diff
@@ -42,7 +41,6 @@ export class PlayerDisplay extends Component {
             this.lookRight = this.lookRight.bind(this);
             this.lookUser = this.lookUser.bind(this);
             this.search = this.search.bind(this);
-            this.searchHistory = this.searchHistory.bind(this);
```
Comment (mmdonaldson) [inline static/src/components/PlayerDisplay.js]: Or the search history functionality

File: static/src/components/PlayerDisplay.js [hunk 3]
```diff
@@ -117,109 +101,118 @@ export class PlayerDisplay extends Component {
             const longCityNameArray = (boardData.wonder).split(' ');
             const city = longCityNameArray.length - 1;
             imageName = longCityNameArray[city].toLowerCase();
+            console.log('--');
```
Comment (mmdonaldson) [inline static/src/components/PlayerDisplay.js]: Can you please remove console.logs

File: static/src/components/PlayerDisplay.js [hunk 4]
```diff
@@ -117,109 +101,118 @@ export class PlayerDisplay extends Component {
...
-                                                                    <Inventory item="ore" amount={boardData.ore + boardData.extra_ore} />
-                                                                    <Inventory item="stone" amount={boardData.stone + boardData.extra_stone} />
-                                                                    <Inventory item="glass" amount={boardData.glass + boardData.extra_glass} />
-                                                                    <Inventory item="paper" amount={boardData.paper + boardData.extra_paper} />
-                                                                    <Inventory item="cloth" amount={boardData.cloth + boardData.extra_cloth} />
-                                                                </List>
-                                                            </TableRowColumn>
-                                                            <TableRowColumn>
-                                                                <CardActions>
-                                                                    { !homeWonder ?
```
Comment (mmdonaldson) [inline static/src/components/PlayerDisplay.js]: Did you mean to delete this?

**Review by mmdonaldson — CHANGES_REQUESTED** [review]
Tests are looking great! Just a few accidental deletions I think, plus a couple of console.logs.

File: static/src/components/PlayerDisplay.js (hunk 4 above)
Comment (wackydapper) [reply static/src/components/PlayerDisplay.js]: Pretty sure i haven't deleted any bit of this render, just got rid of a couple syntax error don't know why its saying i changed the entire table.
Have fixed up those accidental deletions and console log though.

Comment (mmdonaldson) [reply static/src/components/PlayerDisplay.js]: Could be an issue with git but merging this PR will remove this functionality in PlayerDisplay.

Comment (Marthijn-B) [reply static/src/components/PlayerDisplay.js]: This doesn't look like it's been deleted.
lines 135-222 have just been tabbed correctly by the looks
Below the red section is the correctly tabbed green section

**Review by mmdonaldson — APPROVED** [review]

**Review by jacobstringer — CHANGES_REQUESTED** [review]
Make sure files don't get reverted to old versions first. :)

File: static/src/components/Header/index.js [hunk 5]
```diff
@@ -100,7 +99,7 @@ export class Header extends Component {
                                 <Divider />
                                 <MenuItem
                                     id="PlayButton"
-                                    onTouchTap={() => this.dispatchNewRoute('/play')}
+                                    onClick={() => this.dispatchNewRoute('/play')}
```
Comment (mmdonaldson) [inline static/src/components/Header/index.js]: onTouchTap

File: static/src/index.js [hunk 6]
```diff
@@ -1,20 +1,18 @@
-import 'babel-polyfill';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 import { Router, Redirect, browserHistory } from 'react-router';
 import injectTapEventPlugin from 'react-tap-event-plugin';
 import { syncHistoryWithStore } from 'react-router-redux';
 import { polyfill } from 'es6-promise';
-import 'isomorphic-fetch';
```
Comment (mmdonaldson) [inline static/src/index.js]: Need to keep this.

File: static/src/index.js [hunk 7]
```diff
@@ -1,20 +1,18 @@
-import 'babel-polyfill';
```
Comment (mmdonaldson) [inline static/src/index.js:1]: Need to keep this

**Review by mmdonaldson — CHANGES_REQUESTED** [review]
Also need to keep these files
static/src/images/background/capture.jpg
static/src/images/cards/wonderCard.png
static/src/images/icons/wonderCard.png

**Review by mmdonaldson — APPROVED** [review]

**Review by jacobstringer — APPROVED** [review]

**Pull request merged**